modules/ROOT/pages/introduction/intro-config-use-fd.adoc

```diff
@@ -0,0 +1,165 @@
+* Limits the number of reconnection attempts to three at a frequency of every 5000 milliseconds (ms)
++
+.Use the fields in the *Reconnection* section to specify reconnection options.
+image::introduction/intro-config-fd-reconnection-strategy.png[Reconnection strategy example using Flow Designer]
+.. To use TLS, scroll down and specify the required information in the *TLS Configuration* section.
+.. Click *Save*.
+
+For more information about configuring `HTTP Listener`, see the xref:http/http-documentation.adoc[HTTP Connector Reference].
+
+[[scheduler]]
+=== Use Scheduler as a Trigger
+
+Use `Scheduler` to initiate a flow when a time-based condition is met. You can configure `Scheduler` to initiate a flow at a regular interval, or you can configure `Scheduler` to use a more flexible cron expression. A cron expression is a standard for specifying date and time information that initiates a specified action.
+
+To use `Scheduler` as a trigger:
+
+. Follow the procedure described in <<configure-trigger,Configure the Trigger For the Flow>>.
+. In the *Search* field, enter `Scheduler`.
+. Select *Scheduler*.
+. In the *Scheduling Strategy* field, select whether you want to use a fixed frequency or cron expression to trigger the flow:
++
+For example, the following configuration directs `Scheduler` to:
+
+* Wait 10 seconds before initiating the flow for the first time after the app starts
+* Initiate the flow every 30 seconds after the initial 10 second delay
++
+.Select `Fixed Frequency` in the Scheduling Strategy field to initiate a flow at regular intervals.
+image::introduction/intro-config-fd-scheduler-fixed.png[Configuration for a fixed scheduling strategy]
++
+
+The following configuration uses a cron expression to initiate an action every second:
++
+.Select `Cron` in the *Scheduling Strategy* field to use a cron expression to specify when to initiate a flow.
+image::introduction/intro-config-fd-scheduler-cron.png[Configuration using a cron expression]
+
+For more information about configuring `Scheduler`, see xref:mule-runtime::scheduler-concept.adoc[Scheduler Endpoint]
+
+
+[[connector-specific-trigger]]
+=== Use a Connector-Specific Trigger
+
+If a connector has associated triggers, you can use a connector-specific trigger to initiate the flow. For example, if you are configuring an app that contains an Amazon SQS Connector operation, you can use the `Amazon SQS Receive Messages` trigger to initiate a flow each time the configured Amazon SQS queue receives a new message.
+
+To start the configuration for a connector-specific trigger, follow these steps:
+
+. Follow the procedure described in <<configure-trigger,Configure the Trigger For the Flow>>.
+. In the *Search* field, search for and select the connector.
+. If the connector has multiple triggers, select the trigger to use. (If the connector has one trigger, that trigger is selected automatically.)
```

Comment:
Suggest omitting the parenthetical, esp. since it adds a sentence to what should be a single-sentence item.